playground/frontend/README.md

```diff
@@ -21,57 +21,88 @@
 
 ## About
 
-Beam Playground is an interactive environment to try out Beam transforms and examples. The vision for the Playground is to be a web application where users can try out Beam without having to install/initialize a Beam environment.
+Beam Playground is an interactive environment to try out Beam transforms and examples.
+The vision for the Playground is to be a web application where users can try out Beam
+without having to install/initialize a Beam environment.
 
 ## Getting Started
 
 Running, debugging, and testing all require this first step that fetches
-dependencies and generates code:
+dependencies and generates code. Run this in the Beam root:
 
 ```bash
-cd playground_components
-flutter pub get
-flutter pub run build_runner build
-cd ..
-flutter pub get
-flutter pub run build_runner build
```

Comment:
Does this PR still intend on deleting https://github.com/apache/beam/blob/master/playground/frontend/README.md?plain=1#L32-L37?

Reply:
I deleted these lines because with the introduction of symbols extraction the building got more complicated. That is why `:playground:frontend:configure` was introduced.

With the recent idea of keeping generated files under version control, `:playground:frontend:configure` may remain as simple as the above 6 commands. This means it can hardly break, so I guess it should be safe to delete them from the REAMDE. Otherwise we will have two locations to maintain the build commands:
1. `:playground:frontend:configure` Gradle task.
2. README.

Yet the failure of Gradle on your machine with Flutter in path is a concern.

Do you want to keep those commands as a backup?